### PR TITLE
Feature/get ci phpunit working

### PR DESCRIPTION
**Pull request by bart88**
Add simple travis ci config to run phpunit tests against `shepherd` group. Also add that cool badge. This should hopefully encourage everyone to write some tests !

Future things : 
- code coverage 
- code quality 
- more badges 💯

### CODE REVIEW

**Comment by davemikolaj** [comment]
Love your work man - keep fighting the good fight :)

File: .travis.yml [hunk 1]
```diff
@@ -28,5 +28,5 @@ install:
     - composer install
 
 script:
-    - cd $TRAVIS_BUILD_DIR/web
-    - phpunit --group shp --testdox -c ./core ./modules
+    - ./vendor/phpunit/phpunit/phpunit --group shepherd --testdox -c ./web/core ./web/modules
+    - ./vendor/phpunit/phpunit/phpunit --group shepherd -c ./web/core ./web/modules
```
Comment (pingers) [inline .travis.yml]: So, why do we run it twice?

Comment (bart88) [reply .travis.yml]: One pretty one with detailed error log .. yeah I get the point, removing one 🤦‍♂️

**Comment by pingers** [comment]
Is there some travis account configured to run the tests?

**Comment by pingers** [comment]
Nope... just sign in with Github and instant access!

**Pull request merged**